doc/user/content/integrations/monitor/datadog.md

```diff
@@ -0,0 +1,165 @@
+
+```yaml
+...
+connections:
+- "postgres://<USER>:<PASSWORD>@<HOST>:<PORT>/materialize?sslmode=require"
+...
+```
+You can copy this string from the Connect page in your Materialize region.
+
+{{< note >}}
+You need to escape the special `@` character in your user email for a successful connection. For example, `name@email.com`, use `name%40email.com`.
+{{</ note >}}
+
+Create a new app password on the **Connect** page and paste it into the
+`<PASSWORD>` section of the connection command.
+
+Update the second `connections` setting near the end of the file. This
+connection queries your Materialize storage usage and reports on objects using
+storage resources.
+
+Edit the `docker-compose.yaml` file with your Datadog API key and region:
+
+```yaml
+...
+    environment:
+     - DD_API_KEY=<API_KEY> 
+     - DD_SITE=<DATADOG_REGION_ADDRESS>
+...
+```
+
+## Deploy the SQL exporter and Datadog agents
+
+In the current directory, run Docker to deploy the SQL exporter and the Datadog
+agents. The SQL exporter tool uses the connection string you created in a
+previous step to connect with your Materialize instance. 
+
+```shell
+$ docker compose up -d
+```
+
+The Prometheus SQL exporter container runs a service that executes queries against your
+database and then exports the metrics in a format that most monitoring services
+can consume.
+
+The Datadog container runs an agent with account information to communicate
+with the cloud monitoring service. The `docker-compose.yaml` file deploys both
+of these images and configures the Datadog agent with the OpenMetrics
+specification to allow it to parse the metrics returned by the SQL exporter.
+
+## Configure the Datadog dashboard
+
+Now that the containers are running, your Datadog agent will reach out to the
+Datadog endpoint. No metrics are being displayed yet, so let's configure the
+dashboard with some important information.
+
+In the `datadog` directory, copy the contents of the `dashboard.json` file.
+
+In your Datadog instance, navigate to the **Dashboards** section and select 
+**New Dashboard** in the upper right section. Name and create the new dashboard
+then find the gear icon in the upper right navigation. Select **Import dashboard
+JSON** and paste the contents of the `dashboard.json` file.
+
+Once you import, your dashboard will display information
+for your Materialize resources.
+
+## Helpful queries
+
+This dashboard returns a lot of useful information about your Materialize
+resources, but there are some more metrics in the system catalog that you may
+find useful.
+
+In your `config.yml` file, you can add additional metrics to the SQL exporter.
+To display specific information about a table, add  a new query to the
+configuration.
+
+```sql
+queries:
+- name: "total orders"
+   help: "Total Orders"
+   values:
+   - "count"
+   query:  |
+           SELECT count(*) FROM orders
+```
+
+To add more complex queries, you can use the template above and edit the query
```

Comment:
One thing we should warn here is that the user should be cautious about the query. In the sense that a big query can overload the cluster and create issues:

Personal low quality example: _Be cautious when executing big queries, they have the potential to overload the cluster, leading to potential issues._